src/Html.php

```diff
@@ -3534,9 +3536,11 @@ public static function initEditorSystem(
         $enable_images = true,
         int $editor_height = 150,
         array $add_body_classes = [],
+        bool $toolbar = true,
```

Comment:
Please move this new parameter after existing ones, to prevent BC-break on calls that are not using named parameters.